src/content/enterprise/aks.mdx

```diff
@@ -28,9 +28,10 @@ This guide will assume that your domain is registered in [AzureDNS](https://docs
 ### Deploy a Kubernetes cluster
 
 We recommend that you follow Azure's [cluster creation guide](https://docs.microsoft.com/en-us/azure/aks/kubernetes-walkthrough).
+Okteto supports Kubernetes versions 1.20 through 1.22. Support for 1.19 is deprecated.
```

Comment:
Remove the deprecated part. I think is clear enough by saying we support X to Y.

Reply:
although it is marked for deprecation we support v1.19. From what I understand the change would be:
```
Okteto supports Kubernetes versions 1.19 through 1.22.
```
It is right?

Reply:
I thought we no longer support 1.19. @maroshii ?

Reply:
There's confusion about 1.19 support. We are actually taking it into account in the the 0.15.0 testing plan but marked it as deprecated in the release notes.
+1 for "officially" supporting 1.20 1.21 1.22 only (in the docs) and internally supporting 1.19